File: ts-sdk/src/sdk/models/operations/cancelaqueueddeployrequest.ts
import { SpeakeasyBase, SpeakeasyMetadata } from "../../../internal/utils";


export class CancelAQueuedDeployRequestPathParams extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "pathParam, style=simple;explode=false;name=database" })
  database: string;

  @SpeakeasyMetadata({ data: "pathParam, style=simple;explode=false;name=number" })
  number: string;

  @SpeakeasyMetadata({ data: "pathParam, style=simple;explode=false;name=organization" })
  organization: string;
}

export class CancelAQueuedDeployRequestRequest extends SpeakeasyBase {
  @SpeakeasyMetadata()
  pathParams: CancelAQueuedDeployRequestPathParams;
}

export class CancelAQueuedDeployRequestResponse extends SpeakeasyBase {
  @SpeakeasyMetadata()
  contentType: string;

  @SpeakeasyMetadata()
  statusCode: number;

  @SpeakeasyMetadata()
  cancelAQueuedDeployRequest200ApplicationJSONObject?: Record<string, Record<string, any>>;
}

File: go-sdk/pkg/models/operations/cancelaqueueddeployrequest.go
package operations

type CancelAQueuedDeployRequestPathParams struct {
	Database     string `pathParam:"style=simple,explode=false,name=database"`
	Number       string `pathParam:"style=simple,explode=false,name=number"`
	Organization string `pathParam:"style=simple,explode=false,name=organization"`
}

type CancelAQueuedDeployRequestRequest struct {
	PathParams CancelAQueuedDeployRequestPathParams
}

type CancelAQueuedDeployRequestResponse struct {
	ContentType                                        string
	StatusCode                                         int
	CancelAQueuedDeployRequest200ApplicationJSONObject map[string]map[string]interface{}
}
